definitions/divorce/json/CaseEvent/CaseEvent-bailiff-prod.json

```diff
@@ -20,7 +20,7 @@
     "Name": "Draft AoS",
     "Description": "Draft Acknowledgement of Service",
     "DisplayOrder": 1,
-    "PreConditionState(s)": "AosAwaitingSol;AosAwaiting",
+    "PreConditionState(s)": "AosOverdue;AosAwaitingSol;AosAwaiting",
```

Comment:
We have talked to Jeremy, it's ok to add it here since we cannot feature toggle it twice. Once for baillif and for RRJ.

It's jot breaking anything. It's meant to be released soon.

It's a result of testing in https://tools.hmcts.net/jira/browse/DIV-6838.